packages/wonder-blocks-timing/src/util/__tests__/interval.test.js

```diff
@@ -114,12 +124,15 @@ describe("Interval", () => {
 
         it("should invoke setInterval to call the given action", () => {
             // Arrange
+            const intervalSpy = jest.spyOn(global, "setInterval");
             const action = jest.fn();
-            const interval = new Interval(() => action(), 500);
+            const interval = new Interval(
+                () => action(),
+                500,
+                SchedulePolicy.OnDemand,
+            );
```

Comment:
Why the switch to `OnDemand` for this test?

Reply:
Because that was what the test was written to work with, but it didn't use it and so it fails.